crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -11,25 +11,58 @@ use bevy_ecs::prelude::*;
 use bevy_image::Image;
 use bevy_math::{prelude::*, FloatExt, FloatOrd};
 use bevy_picking::backend::prelude::*;
+use bevy_reflect::prelude::*;
 use bevy_render::prelude::*;
 use bevy_transform::prelude::*;
 use bevy_window::PrimaryWindow;
 
+/// How should the [`SpritePickingPlugin`] handle picking and how should it handle transparent pixels
+#[derive(Debug, Clone, Copy, Reflect)]
+pub enum SpritePickingMode {
```

Comment:
Maybe we should add a None type here. Today we have a feature flag for enabling or disabling sprite picking. This makes it difficult to undo that (for example, I wanted to copy this picking behavior and had to disable all the default features and re-enable everything but sprite picking in order to do so).

Reply:
I'm happy to go with the general opinion here but I feel like that probably falls outside the scope of this change into a wider conversation about uses of feature flags right?

If you want to disable this change, going back to the old style then `BoundingBox` already does that.

Reply:
Yeah, let's leave that for another PR :)